packages/apm/src/hubextensions.ts

```diff
@@ -26,43 +27,50 @@ function traceHeaders(): { [key: string]: string } {
  *
  * @param spanContext Properties with which the span should be created
  */
-function startSpan(spanContext?: SpanContext): Span {
+function startSpan(spanOrTransactionContext: SpanContext | TransactionContext): Transaction | Span {
```

Comment:
I think we can just name it `context`, `spanOrTransactionContext` sounds excessive. Especially that we have types in place.